modules/Bio/EnsEMBL/ProteinFeature.pm

```diff
@@ -82,21 +83,33 @@ use vars qw(@ISA);
 
 =cut
 
+
 sub new {
   my $proto = shift;
```

Comment:
It's recommended to always unpack `@_` first.
this means:
`
 sub new {
   my ($proto, @args) =   @_;
`
and from there you can use `@args` and leave the `@_` alone.